### DISCUSSION TITLE
Possible to get height back from element

### DISCUSSION THREAD

**Discussion by Kazagan** [QuestPDF/QuestPDF#617] [Q&A] [answered]
When generating the IContainer for an element, is it possible to get the height back from the element? I tried calculating it might self, based on padding and font sizes, but found that I was off by quite a bit. Trying to use the Height to enable some manual cut off points for content, so that I can pass it to a "continued" , but need access to the header and footer heights to do so.

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We apologize for the long delay. This is exactly what Dynamic Components are designed for. Their Compose method runs per page with access to the available space, and any element you build exposes its measured Size.Height — including your header and footer — so you can compute the remaining space and decide where to cut content off into a "Continued" block. The "table with per-page subtotals" example measures header and footer heights to do precisely this:
https://www.questpdf.com/concepts/code-patterns/dynamic-components.html

We are closing this thread as resolved, but should you run into any further problems, please don't hesitate to open a new issue.

Thank you!
